apps/site/src/pages/wordpress.page.tsx

```diff
@@ -0,0 +1,18 @@
+import { FunctionComponent } from "react";
+
+import { Availability } from "../components/pages/wordpress/availability";
+import { EmailSubmittedProvider } from "../components/pages/wordpress/email-submitted-context";
+import { Header } from "../components/pages/wordpress/header";
+import { WhyDevelop } from "../components/pages/wordpress/why-develop";
+import { WhyInstall } from "../components/pages/wordpress/why-install";
+
+const WordPressPage: FunctionComponent = () => (
+  <EmailSubmittedProvider>
+    <Header />
```

Comment:
@luisbettencourt Can we set a max width on the body column with the content in? At super-wide screen sizes (e.g. 2560px on a 5k monitor) things look pretty stretched.

Things like the helix background can continue to stretch to fill the page as they are now.

Reply:
done in d64666d6ed5f3165c1deed829738c0726b528391